### ISSUE TITLE
Eclipse OpenJ9 needs to use the trademark symbol

### ISSUE THREAD

**Issue by pshipton** [eclipse-openj9/openj9-docs#986]
There are some requirements for using the trademark symbol ™ in the Eclipse OpenJ9 release review.
i.e. Eclipse OpenJ9™

https://gitlab.eclipse.org/eclipsefdn/emo-team/emo/-/issues/368

I don't this used in the User Guide or Web site. We may need to fix this asap to pass the release review.

**Comment by pshipton** [comment]
I see this guidance in the Eclipse issue.

- The formal name, e.g. "Eclipse Foo™", is used in the project title
- The formal name including appropriate marks (e.g, "™") is used in the first mention in the text of the project description, and scope
- Project website uses the [project's formal name](https://www.eclipse.org/projects/handbook/#trademarks-website-name) in first and all prominent references

**Comment by Sreekala-Gopakumar** [comment]
I will update the documents. Thanks!

**Comment by doveye** [comment]
I can't find any record of "Eclipse OpenJ9" being a trademarked term, and it doesn't come up in a search of the US trademark database (https://www.uspto.gov/trademarks/search). @pshipton is there a record somewhere of this name actually being trademarked?

**Comment by doveye** [comment]
Thanks Peter. Yes I wondered about the project page. I found the bug for the original trademark request at https://bugs.eclipse.org/bugs/show_bug.cgi?id=519050 and a list of trademarks (including Eclipse OpenJ9) at https://www.eclipse.org/legal/trademarks.php, which I think covers it. I also found that you don't have to register a trademark, so I guess that explains why it's not in the USPTO database (https://www.uspto.gov/trademarks/basics/why-register-your-trademark).
